seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
```

Comment:
Is there a reason seleniumDescriptors is a separate module rather than included in the selenium tunnel?